api/runner/async_runner.go

```diff
@@ -52,8 +51,7 @@ func getCfg(task *models.Task) *Config {
 		Timeout: time.Duration(*task.Timeout) * time.Second,
 		ID:      task.ID,
 		AppName: task.AppName,
-		Stdout:  stderr,
-		Stderr:  stderr,
+		Stdout:  ioutil.Discard,
```

Comment:
Is the intention really to discard Stdout? I understand that stderr is being plugged later down the process, but stdout not.

`getCfg()` is used by `startAsyncRunners()` in order to connect both stdout and stderr. Could you please check that this behavior hasn't been changed?